eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -63,3 +64,17 @@ steps:
       -PRBranch "${{ parameters.PRBranchName }}"
       -AuthToken "$(azuresdk-github-pat)"
       -PRTitle "${{ parameters.PRTitle }}"
+
+
+- task: PowerShell@2
```

Comment:
We should probably have a condition around this step if the reviewer isn't set we should skip or exclude this step.

Reply:
Yeah. We have the complication is that we don't know the variable

I do have a commit floating around earlier that did this:

```
 - ${{ if or(gt(length(parameters.GithubReviewers), 0),gt(length(parameters.GithubTeamReviewers), 0)) }}:
``` 

But it's essentially useless because parameters resolve at template compile time, not runtime. This means that the variable that I set won't be available. An alternative is to simply place the **variable name** as the argument, then I can access this like `variables["blah"]` and have a chance of it resolving.

Reply:
Alrighty this is implemented now. I'm testing it with a pipeline.

Reply:
That testing was just a barrel of fun. Finally got something working the way I expect [over here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=450521&view=logs&s=920b375f-847c-504d-8cee-9a1d03ea4bae).

I'll comment the PR where I expect to replace a methodology.